akka-bench-jmh/src/main/scala/akka/actor/BenchmarkActors.scala

```diff
@@ -98,7 +98,7 @@ object BenchmarkActors {
     val fullPathToDispatcher = "akka.actor." + dispatcher
     val latch = new CountDownLatch(numPairs * 2)
     val actors = for {
-      i <- (1 to numPairs).toVector
+      _ <- (1 to numPairs).toVector
```

Comment:
Can this also be
```
List.fill(numPairs) {
  val ping = system.actorOf(PingPong.props(messagesPerPair, latch).withDispatcher(fullPathToDispatcher))
  val pong = system.actorOf(PingPong.props(messagesPerPair, latch).withDispatcher(fullPathToDispatcher))
  (ping, pong)
}
```
... but this is OK as well :D

Reply:
👍 updating